lib/Listener/BeforeTemplateRenderedListener.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2023 T-Systems International
+ *
+ * @author M. Mura <mauro-efisio.mura@t-systems.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCA\NmcSpica\Listener;
+
+use OCP\AppFramework\Http\Events\BeforeTemplateRenderedEvent;
+use OCP\AppFramework\Http\TemplateResponse;
+use OCP\EventDispatcher\Event;
+use OCP\EventDispatcher\IEventListener;
+use OCP\IUserSession;
+
+class BeforeTemplateRenderedListener implements IEventListener {
+	private IUserSession $userSession;
+
+	public function __construct(IUserSession $userSession) {
+		$this->userSession = $userSession;
+	}
+
+	public function handle(Event $event): void {
+		$user = $this->userSession->getUser();
+
+		if ($user === null) {
+			return;
+		}
```

Comment:
Nitpick: This is kind of duplicate with the `isLoggedIn()` return value of the event

https://github.com/nextcloud/server/blob/f3ae617bde99f8babddca634edb1f1f632b69191/lib/private/AppFramework/Middleware/AdditionalScriptsMiddleware.php#L52
https://github.com/nextcloud/server/blob/master/lib/private/User/Session.php#L269-L275

Reply:
Hi! You're absolutely right. I removed the redundancy.